### PR TITLE
1569 add travis support

### PR DESCRIPTION
**Pull request by splekhanov**

### CODE REVIEW

**Comment by codacy-review** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/app/42_/jdi-light/pullRequest?prid=4714033&bid=15744444)

File: jdi-light/src/main/java/com/epam/jdi/light/driver/get/DriverData.java [hunk 1]
```diff
@@ -57,6 +58,7 @@ public static String getDriverFolder() {
     public static String BROWSER_SIZE = "MAXIMIZE";
     public static final String DEFAULT_DRIVER = "chrome";
     public static String DRIVER_NAME = DEFAULT_DRIVER;
+    public static String ARGUMENTS_PROPERTY = "arguments";
```
Comment (codacy-review) [inline jdi-light/src/main/java/com/epam/jdi/light/driver/get/DriverData.java]: Codacy Issue found: [Fields should be declared at the top of the class, before any method declarations, constructors, initializers or inner classes.](https://app.codacy.com/app/42_/jdi-light/pullRequest?prid=4714033)

**Comment by codacy-review** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 4
           
```


See the complete overview on [Codacy](https://app.codacy.com/app/42_/jdi-light/pullRequest?prid=4714033&bid=15744444)

File: jdi-light/src/main/java/com/epam/jdi/light/driver/get/DriverData.java (hunk 1 above)
Comment (codacy-review) [inline jdi-light/src/main/java/com/epam/jdi/light/driver/get/DriverData.java]: Codacy Issue found: [Fields should be declared at the top of the class, before any method declarations, constructors, initializers or inner classes.](https://app.codacy.com/app/42_/jdi-light/pullRequest?prid=4714033)

File: jdi-light/src/main/java/com/epam/jdi/light/settings/WebSettings.java [hunk 2]
```diff
@@ -17,6 +17,9 @@
 import org.openqa.selenium.WebDriver;
 import org.openqa.selenium.WebElement;
 
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.IOException;
```
Comment (codacy-review) [inline jdi-light/src/main/java/com/epam/jdi/light/settings/WebSettings.java]: Codacy Issue found: [Avoid unused imports such as 'java.io.IOException'](https://app.codacy.com/app/42_/jdi-light/pullRequest?prid=4714033)

File: jdi-light/src/main/java/com/epam/jdi/light/settings/WebSettings.java [hunk 3]
```diff
@@ -17,6 +17,9 @@
 import org.openqa.selenium.WebDriver;
 import org.openqa.selenium.WebElement;
 
+import java.io.File;
```
Comment (codacy-review) [inline jdi-light/src/main/java/com/epam/jdi/light/settings/WebSettings.java]: Codacy Issue found: [Avoid unused imports such as 'java.io.File'](https://app.codacy.com/app/42_/jdi-light/pullRequest?prid=4714033)

File: jdi-light/src/main/java/com/epam/jdi/light/settings/WebSettings.java [hunk 4]
```diff
@@ -17,6 +17,9 @@
...
+import java.io.FileInputStream;
```
Comment (codacy-review) [inline jdi-light/src/main/java/com/epam/jdi/light/settings/WebSettings.java]: Codacy Issue found: [Avoid unused imports such as 'java.io.FileInputStream'](https://app.codacy.com/app/42_/jdi-light/pullRequest?prid=4714033)

**Comment by mf-code** [comment]
[4282] - https://transfer.sh/vc3yB/jdi-light-html-tests.tar.gz

**Comment by mf-code** [comment]
[4282] - https://transfer.sh/msXmN/jdi-light-examples.tar.gz

**Comment by mf-code** [comment]
[4282] - https://transfer.sh/gWxQa/jdi-performance.tar.gz

**Comment by mf-code** [comment]
[4282] - Allure report on Netlify: https://5dfcae29e91cbabe1af9139f--musing-swartz-6fc060.netlify.com

**Comment by mf-code** [comment]
[4284] - https://transfer.sh/zjOQV/jdi-light-html-tests.tar.gz

**Comment by mf-code** [comment]
[4284] - https://transfer.sh/7pkiW/jdi-performance.tar.gz

File: jdi-light/src/main/java/com/epam/jdi/light/driver/get/DriverData.java (hunk 1 above)
Comment (codacy-review) [inline jdi-light/src/main/java/com/epam/jdi/light/driver/get/DriverData.java]: Codacy Issue found: [Fields should be declared at the top of the class, before any method declarations, constructors, initializers or inner classes.](https://app.codacy.com/app/42_/jdi-light/pullRequest?prid=4714033)

**Comment by codacy-review** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/app/42_/jdi-light/pullRequest?prid=4714033&bid=15744444)

**Comment by mf-code** [comment]
[4284] - https://transfer.sh/Anx43/jdi-light-examples.tar.gz

**Comment by mf-code** [comment]
[4284] - Allure report on Netlify: https://5dfcb37b738b6dea4d71053b--musing-swartz-6fc060.netlify.com

**Comment by mf-code** [comment]
[4288] - https://transfer.sh/8QdXt/jdi-light-html-tests.tar.gz

**Comment by mf-code** [comment]
[4288] - https://transfer.sh/LYEEA/jdi-performance.tar.gz

**Comment by mf-code** [comment]
[4288] - https://transfer.sh/3ihjQ/jdi-light-examples.tar.gz

**Comment by mf-code** [comment]
[4288] - Allure report on Netlify: https://5dfcb6dec739e9a24841e24f--musing-swartz-6fc060.netlify.com

**Comment by codacy-review** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/app/42_/jdi-light/pullRequest?prid=4714033&bid=15744444)

File: jdi-light/src/main/java/com/epam/jdi/light/driver/get/DriverData.java (hunk 1 above)
Comment (codacy-review) [inline jdi-light/src/main/java/com/epam/jdi/light/driver/get/DriverData.java]: Codacy Issue found: [Fields should be declared at the top of the class, before any method declarations, constructors, initializers or inner classes.](https://app.codacy.com/app/42_/jdi-light/pullRequest?prid=4714033)

**Comment by mf-code** [comment]
[4294] - https://transfer.sh/Jzit0/jdi-performance.tar.gz

**Comment by mf-code** [comment]
[4294] - https://transfer.sh/8z43O/jdi-light-examples.tar.gz

**Comment by mf-code** [comment]
[4294] - https://transfer.sh/hmgy3/jdi-light-html-tests.tar.gz

**Comment by mf-code** [comment]
[4294] - Allure report on Netlify: https://5dfcc79de9ce84030b2ae23e--musing-swartz-6fc060.netlify.com

**Comment by mf-code** [comment]
[4300] - https://transfer.sh/AXrzq/jdi-light-examples.tar.gz

**Comment by mf-code** [comment]
[4300] - https://transfer.sh/S4uEm/jdi-performance.tar.gz

**Comment by mf-code** [comment]
[4300] - https://transfer.sh/tvkOV/jdi-light-html-tests.tar.gz

**Comment by mf-code** [comment]
[4300] - Allure report on Netlify: https://5dfcde2a4e46f0c0b7664f58--musing-swartz-6fc060.netlify.com

**Comment by codacy-review** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=4714033&bid=15744444)

File: jdi-light/src/main/java/com/epam/jdi/light/driver/get/DriverData.java [hunk 5]
```diff
@@ -57,6 +57,7 @@ public static String getDriverFolder() {
...
```
Comment (codacy-review) [inline jdi-light/src/main/java/com/epam/jdi/light/driver/get/DriverData.java]: Codacy Issue found: [Fields should be declared at the top of the class, before any method declarations, constructors, initializers or inner classes.](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=4714033)

**Comment by mf-code** [comment]
[4447] - https://transfer.sh/q2XsY/jdi-light-examples.tar.gz

**Comment by mf-code** [comment]
[4447] - https://transfer.sh/yvLB1/jdi-light-html-tests.tar.gz

**Comment by mf-code** [comment]
[4447] - https://transfer.sh/XZj2h/jdi-performance.tar.gz

**Comment by mf-code** [comment]
[4447] - Allure report on Netlify: https://5e1c732cdf7f258e39421e1c--musing-swartz-6fc060.netlify.com

**Comment by codacy-review** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=4714033&bid=15744444)

File: jdi-light/src/main/java/com/epam/jdi/light/driver/get/DriverData.java (hunk 5 above)
Comment (codacy-review) [inline jdi-light/src/main/java/com/epam/jdi/light/driver/get/DriverData.java]: Codacy Issue found: [Fields should be declared at the top of the class, before any method declarations, constructors, initializers or inner classes.](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=4714033)

**Comment by mf-code** [comment]
[4478] - Could not save metadata Could not save metadata Could not save metadata

**Comment by mf-code** [comment]
[4478] - Could not save metadata Could not save metadata Could not save metadata

**Comment by mf-code** [comment]
[4478] - Could not save metadata Could not save metadata Could not save metadata

**Pull request merged**